docs/partners/plan-provisioning.md

```diff
@@ -29,7 +29,9 @@ We like to think that integrating with Jetpack Start is fairly easy. From beginn
 2. Ensure Jetpack is installed on the WordPress site:
     - `wp plugin install jetpack`
 3. Run the following script with the Jetpack Partner ID and token that were provided to you
-    - `./wp-content/plugins/jetpack/bin/partner-provision.sh --partner_id={partner_id} --partner_secret={partner_secret} --user_id={wordpress_user_id} --plan={plan_slug} [--url=http://example.com]`
+    - `./wp-content/plugins/jetpack/bin/partner-provision.sh --partner_id={partner_id} --partner_secret={partner_secret} --user_id={user_id_on_wp_installation_at_partner} --plan={plan_slug} [--url=http://example.com]`
+        - Note: The `--user_id` argument here is the ID of the user on the WordPress installation at the host. For example, on a new install, this value will likely be `1`.
+        - Tip: If you don't have the user ID, but do have the user's email, you should be able to do something like this: `--user_id=$( wp user get {email} --field=ID )`
```

Comment:
I think a better solution would be to have a separate parameter to pass to `partner-provision.sh`, such as `--user_email`. It's less messy and would allow for catching errors if a user couldn't be found with that email address. I think weird things would happen with your example if a user couldn't be found.